lib/index.js

```diff
@@ -0,0 +1,72 @@
+/*
+ * Bedrock Agreement HTTP Module.
+ *
+ * Copyright (c) 2016 Digital Bazaar, Inc. All rights reserved.
+ */
+/* jshint node: true */
+var bedrock = require('bedrock');
+var brAgreement = require('bedrock-agreement');
+var brRest = require('bedrock-rest');
+var brPassport = require('bedrock-passport');
+var validate = require('bedrock-validation').validate;
+
+var BedrockError = bedrock.util.BedrockError;
+var ensureAuthenticated = brPassport.ensureAuthenticated;
+
+// load config defaults
+require('./config');
+
+bedrock.events.on('bedrock.test.configure', function() {
+  require('./test.config');
+});
+
+// add routes
+bedrock.events.on('bedrock-express.configure.routes', addRoutes);
+
+function addRoutes(app) {
+  var basePath = bedrock.config['bedrock-agreement-http'].basePath;
+
+  // get agreements for the authenticated identity
+  app.get(
+    basePath, ensureAuthenticated, brRest.when.prefers.ld,
+    brRest.makeResourceHandler({
+      get: function(req, res, callback) {
+        var actor = req.user ? req.user.identity : undefined;
+        brAgreement.get(actor, callback);
+      }
+    }));
+
+  // create a new agreement for the authenticated identity
+  app.post(
+    basePath, ensureAuthenticated, validate('services.agreement.postAgreement'),
```

Comment:
Use `.routes.accepted` instead of `basePath` to accept an agreement. In the future, we may want to post to `/agreements` (`basePath`) to add actual agreements via this HTTP API.